sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -1023,4 +1023,26 @@ class CastSuite extends SparkFunSuite with ExpressionEvalHelper {
       checkEvaluation(ret, InternalRow(null))
     }
   }
+
+  test("SPARK-27768 process infinity, -infinity , nan in case insensitive manner") {
```

Comment:
Actually, this is not a bug. Instead, this is a behavior change for PostgreSQL compatibility.
Previously, Spark works like Java/Scala/Hive.

Reply:
@dongjoon-hyun OK.. will remove the JIRA id.